### ISSUE TITLE
Via stitching error in KiCad 8.99 Nightly

### ISSUE THREAD

**Issue by VoxelTek** [jsreynaud/kicad-action-scripts#75]
Hi there,

When trying to use Via stitching in KiCad 8.99, the nightly release, I get this error:

![image](https://github.com/jsreynaud/kicad-action-scripts/assets/53562267/e5e517ee-491c-4b34-b898-9bf7519750bd)

This is the version I am using:
![image](https://github.com/jsreynaud/kicad-action-scripts/assets/53562267/ebcc9097-f21b-43ab-86d2-5c325a87a873)

Here is a video demonstrating the issue:
https://github.com/jsreynaud/kicad-action-scripts/assets/53562267/9223eedd-d8bb-4816-ad01-7a410bf37b15

Debug mode does not appear to help, as shown in the video.

I would greatly appreciate any assistance given regarding this error.

Kind regards,
@VoxelTek

**Comment by VoxelTek** [comment]
Update:

I modified `FillArea.py`, and changed line 602 from:
```py
area_clearance = area.GetLocalClearance()
```
to:
```py
area_clearance = int(area.GetLocalClearance())
```

This solved the errors I was having previously, but now, I get this error:

![image](https://github.com/jsreynaud/kicad-action-scripts/assets/53562267/fb217427-61c3-4d14-af10-17b0268172e0)

**Comment by jsreynaud** [comment]
Yes, there are some missing feature in dev branch of KiCad that avoid to have this plugging working for the moment.
Once KiCad will provide needed functions I'll update it....
